velox/common/file/FileSystems.h

```diff
@@ -87,6 +91,11 @@ class FileSystem {
   /// the folder.
   virtual std::vector<std::string> list(std::string_view path) = 0;
 
+  /// Returns all the folders in a path recursively.
+  virtual std::vector<std::string> listFolders(std::string_view path) {
```

Comment:
Let's implement this in fuzzer test not file system api such as velox/common/file/Utils.h?